### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by mruminski**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+        "indent": [
+            "error",
+            4
```
Comment (matwjablonski) [inline .eslintrc.js:21]: here is your declared indentation ;) if you want to have 2 spaces you need to change this line ;)

File: views/index.pug [hunk 2]
```diff
@@ -0,0 +1,16 @@
+doctype html
+html
```
Comment (matwjablonski) [inline views/index.pug]: lang

File: views/index.pug [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  head
+    title="Mobi reader"
```
Comment (matwjablonski) [inline views/index.pug:5]: meta

File: views/logged.pug [hunk 4]
```diff
@@ -0,0 +1,8 @@
+doctype html
+html
```
Comment (matwjablonski) [inline views/logged.pug]: as above

File: .eslintrc.js (hunk 1 above)
Comment (mruminski) [reply .eslintrc.js:21]: Thanks :)

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**